packages/plugin-constraints/sources/commands/constraints/query.ts

```diff
@@ -7,7 +7,7 @@ import {Constraints}            from '../../Constraints';
 
 // eslint-disable-next-line arca/no-default-export
 export default class ConstraintsQueryCommand extends BaseCommand {
-  @Command.Boolean(`--json`)
+  @Command.Boolean(`--json`, {description: `Follow a JSON-stream output also known as NDJSON (https://github.com/ndjson/ndjson-spec)`})
```

Comment:
For all the `--json` options, I think it's overly verbose to add the URL in a one-line description.

```suggestion
  @Command.Boolean(`--json`, {description: `Formats the output using an NDJSON stream`})
```